### PR TITLE
Real Time Translation: Main Quest from Veja Ravine to Kvailas Forest, + Fedimian villagers + Mage Tower

### PR DESCRIPTION
**Pull request by vtange**
Reupload of lines done during iCBT.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -1198,12 +1198,12 @@ QUEST_LV_0100_20150317_001197	Stealth, detection, trap installation. {nl} Scouts
 QUEST_LV_0100_20150317_001198	We Sadhu are always on an eternal journey. {nl}When our journeys in this world come to an end, we'll still continue in the afterlife.
 QUEST_LV_0100_20150317_001199	Nothing is unfair in real battles.{nl}You've gotta do anything to win. Don't you think so?
 QUEST_LV_0100_20150317_001200	We should find it quickly..{nl}It may be an end to Macians' Tower.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Macians' Tower -> Mage Tower

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -1432,19 +1432,19 @@ QUEST_LV_0100_20150317_001431	There is Fedimian, but there are interesting ruins
...
 QUEST_LV_0100_20150317_001434	It was the place where people lived and supported the previous rebel Ruklys. {nl} the curse would be hard while you went so still valid.
-QUEST_LV_0100_20150317_001435	The Goddess Zemynas Quezon and Irasshai me always Jofuiru protect. {nl} brother also I believe that is the Mina has received a care.
-QUEST_LV_0100_20150317_001436	How many of the Goddess if to be increased a prayer to the Goddess Zemyna Will He will hear the prayer?
-QUEST_LV_0100_20150317_001437	Here, Chengdu Fedimian God, is the Earth. Please ask {nl} commercial district is Irasshara as it entered it because it is currently in closed.
-QUEST_LV_0100_20150317_001438	The divine city, Fedimian also has business district besides divine district here.{nl}But, on Medis Diena, business district was completly destroyed due to the earthquake.
-QUEST_LV_0100_20150317_001439	We are still reviving the city, but since it's too old..{nl}At least, divine district is still left due to Gemina's blessing.
+QUEST_LV_0100_20150317_001435	The Goddess Zemynas has always protected Jofuiru for me. {nl} I believe my brother also leaves me to Zemynas' care.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: The Goddess Zemynas -> Goddess Zemyna

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -1432,19 +1432,19 @@ QUEST_LV_0100_20150317_001431	There is Fedimian, but there are interesting ruins
...
+QUEST_LV_0100_20150317_001443	If that vicious atmosphere spreads to other forests..{nl}Those forests will also change, like Kvailas Forest.{nl}That this could happen to the whole kingdom.. it scares me..
 QUEST_LV_0100_20150317_001444	Follower Evaldas
-QUEST_LV_0100_20150317_001445	Everyone is tired and the vicious atmosphere never stops so we should at least try to work happily. {nl}However, I don't know how long I would try to feel happy.
+QUEST_LV_0100_20150317_001445	Everyone is tired, but the vicious atmosphere never stops, so we at least try to stay happy {nl} working. However, I don't know how long I can keep feeling happy.
 QUEST_LV_0100_20150317_001446	Follower Zanetta
-QUEST_LV_0100_20150317_001447	I want to tell to the people out there that we are still here.{nl}But, the Goddess is keep telling us that everything will be alright when the revelation comes.
+QUEST_LV_0100_20150317_001447	I want to tell to the people out there that we are still here.{nl}The goddess always tells us that everything will be alright when the revelation comes.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: The goddess -> The Goddess

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -6943,35 +6943,35 @@ QUEST_LV_0100_20150714_006942	Hmm. You're really good as they say. {nl}Rolandas
...
+QUEST_LV_0100_20150714_006961	Help me. Infroholders are chasing after me.{nl}I will lose when they gang up on me.
 QUEST_LV_0100_20150714_006962	Put Matsum's flower alcohol in this potion and shake it. {nl} It will help you withstand the muddy aura of Bramble.
 QUEST_LV_0100_20150714_006963	This place is too dangerous. {nl}Please go to a safe area.
 QUEST_LV_0100_20150714_006964	I can feel Clymen in Ishpirki Entrance. {nl}Hurry and get rid of it.
-QUEST_LV_0100_20150714_006965	Bramble...{nl}It is so pity that it denys its fate.{nl}
+QUEST_LV_0100_20150714_006965	Bramble…{nl}It is a pity that it denies its fate.{nl}
 QUEST_LV_0100_20150714_006966	When I gained my conciousness, a few hundreds of years have passed.
 QUEST_LV_0100_20150714_006967	Jane's spirit
-QUEST_LV_0100_20150714_006968	I saved nothing.{nl}Master, Magicians' Tower, and my peace...{nl}
+QUEST_LV_0100_20150714_006968	I saved nothing.{nl}Master, Magicians' Tower, and my peace…{nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Magicians' Tower -> Mage Tower

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -7576,16 +7576,16 @@ QUEST_LV_0100_20150714_007575	Absorb the grudge into the beads of comfort
...
 QUEST_LV_0100_20150714_007577	The resentment of the villagers in Andail Village became negative aura and it is floating around. Gather the resentment and fill it into the beads of comfort.
 QUEST_LV_0100_20150714_007578	You've filled the resentment of the villagers in Andail Village into the beads. Hand them over to Bokor Master.
-QUEST_LV_0100_20150714_007579	The request of Saule Goddess
-QUEST_LV_0100_20150714_007580	Saule Goddess needs your help . Talk to her.
+QUEST_LV_0100_20150714_007579	The request of Goddess Saule
+QUEST_LV_0100_20150714_007580	Goddess Saule needs your help . Talk to her.
 QUEST_LV_0100_20150714_007581	Eliminate the thorny vines that are on the way to Gate Route
 QUEST_LV_0100_20150714_007582	Saule is worried that Bramble's thorny vines may spread to Septyni Glen. Eliminate the thorny vines that are on the way to Gate Route.
-QUEST_LV_0100_20150714_007583	You've cut down the threatening thorny vines. Report to Saule Goddess.
+QUEST_LV_0100_20150714_007583	You've cut down the threatening thorny vines. Report to Goddess Saule.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Report to Goddess Saule. -> Report back to Goddess Saule.

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -7576,16 +7576,16 @@ QUEST_LV_0100_20150714_007575	Absorb the grudge into the beads of comfort
...
 QUEST_LV_0100_20150714_007584	Eliminate the thorny vines
-QUEST_LV_0100_20150714_007585	Saule Goddess wants purify the rest of Thorny Forest. Talk to Saule Goddess.
-QUEST_LV_0100_20150714_007586	Set the crystal of the authority at Sirudgellar Forest
-QUEST_LV_0100_20150714_007587	Saule Goddess wants to purify the Thorny Forest with the crystal that contains her authority. Put the crystal of the authority at Sirudgellar Forest.
-QUEST_LV_0100_20150714_007588	You've set the crystal that possesses the authority of Saule Goddess. Go back to Saule Goddess.
+QUEST_LV_0100_20150714_007585	Goddess Saule wants purify the rest of Thorny Forest. Talk to Goddess Saule.
+QUEST_LV_0100_20150714_007586	Set the crystal of the authority at Sirdgela Forest
+QUEST_LV_0100_20150714_007587	Goddess Saule wants to purify the Thorny Forest with the crystal that contains her authority. Put the crystal of the authority at Sirdgela Forest.
+QUEST_LV_0100_20150714_007588	You've set the crystal that possesses the authority of Goddess Saule. Go back to Goddess Saule.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Go back to Goddess Saule. -> Return to Goddess Saule.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**